uaclient/apt.py

```diff
@@ -340,6 +340,21 @@ def run_apt_update_command(
     return out
 
 
+def update_sources_list(sources_list: str):
+    cache = get_apt_cache()
+    try:
+        util.retry(
+            (apt.cache.LockFailedException, apt.cache.FetchFailedException),
+            APT_RETRIES,
+        )(cache.update)(sources_list=sources_list)
+    except apt.cache.LockFailedException:
```

Comment:
For readability's sake, I suggest that we can wrap the outer function instead.
Feel free to disagree though, works well as is.

```suggestion
@util.retry(
    (exceptions.APTProcessConflictError, exceptions.APTUpdateFailed),
    APT_RETRIES,
)
def update_sources_list(sources_list: str):
    cache = get_apt_cache()
    try:
        cache.update(sources_list=sources_list)
    except apt.cache.LockFailedException:
```
`(...)`

Reply:
Addressed, thanks.